script.js

```diff
@@ -0,0 +1,37 @@
+// Initializing an array for storing the quiz questions
+let quizQuestions = [];
+
+// Function to create an object from the quiz question
+document
+    .getElementById("quiz-form")
+    .addEventListener("submit", function (event) {
+        // To prevent refreshing the page after submitting
+        event.preventDefault();
+
+        const question = document.getElementById("question").value;
+        const options = [
+            document.getElementById("option1").value,
+            document.getElementById("option2").value,
+            document.getElementById("option3").value,
+            document.getElementById("option4").value,
+        ];
```

Comment:
You could also get all the option in your HTML file by doing:

`const allOptions = document.querySelectorAll('option');`

And then if you want the values you could loop over the `allOptions`.

This would be preferred on bigger products that you may want to scale in the future. This what it isn't dependent on you specifically declaring option1,2,3,4 each time

Reply:
const allOptions = document.querySelectorAll('option')
 does it mean I have to have <option> instead of <input> in my HTML file to select by querySelector?

Reply:
```
const optionInputs = document.querySelectorAll(".option-container input[type='text']");
const options = Array.from(optionInputs).map((input) => input.value);
```
I could not use const allOptions = document.querySelectorAll('option')
so I went the other way